content/en/tracing/manual_instrumentation/ruby.md

```diff
@@ -1,23 +1,130 @@
 ---
-title: Ruby Manual Instrumentation
+title: Ruby Custom Instrumentation
 kind: documentation
-decription: 'Manually instrument your Ruby application to send custom traces to Datadog.'
+aliases:
+    - /tracing/opentracing/ruby
+description: 'Implement the OpenTracing standard with the Datadog Java APM tracer.'
```

Comment:
I think you meant Ruby, not Java. Also this isn't OpenTracing?

Reply:
Ah yes I've been lax about changing these. Will update